app-android/src/main/java/com/ustadmobile/port/android/impl/UstadApp.kt

```diff
@@ -122,6 +126,40 @@ open class UstadApp : BaseUstadApp(), DIAware {
                 it.setupAssignmentSyncListener(context, di)
             }
         }
+        bind<File>(tag = DiTag.TAG_TORRENT_DIR) with scoped(EndpointScope.Default).singleton{
+            val torrentDir = File(applicationContext.filesDir.siteDataSubDir(context), "torrent")
```

Comment:
There should be a constant for the torrent dir name instead of using a plain string for "torrent"